cirq/_compat.py

```diff
@@ -22,10 +22,32 @@
 # sign of the second argument.
 
 # coverage: ignore
+from typing import Any
+
+import sympy
+import numpy as np
+
+
 if sys.version_info < (3,):
     import fractions   # pylint: disable=unused-import
 
     def gcd(a, b):
         return abs(fractions.gcd(a, b))
 else:
     from math import gcd  # pylint: disable=unused-import
+
+
+def proper_repr(value: Any) -> str:
+    """Overrides a few cases that return strings that don't parse."""
```

Comment:
I'd add a list of these cases to doc string.

Reply:
Done.